package.json

```diff
@@ -28,13 +28,13 @@
   "scripts": {
     "analyze": "cross-env ANALYZE_MODE=true NODE_ENV=production webpack",
     "browserstack": "node e2e/browserstack.js -c e2e/browserstack.conf.js -e browserstack ",
-    "build": "run-s clean getcapabilities build:config build:dev",
+    "build": "run-s clean build:config build:dev",
     "build:dev": "cross-env NODE_ENV=development webpack",
-    "build:config": "bash -c 'tasks/buildOptions.sh' && node ./tasks/config.js",
+    "build:config": "run-s getcapabilities && bash -c 'tasks/buildOptions.sh' && node ./tasks/config.js",
```

Comment:
I actually use `build:config` quite a bit to avoid the `getcapabilites` requests when I am editing config files or build tasks.  Maybe we could break this out into another task like `build:ci` or `build:travis`?

Reply:
I was wondering about that use case...  Sounds good, I'll update this!